README

```diff
@@ -313,8 +308,8 @@ Required for DreamPlug:
 
 Now, fetch the git source of freedom-maker and build the image you want:
 
-    $ git clone http://anonscm.debian.org/git/freedombox/freedom-maker.git freedom-maker
-    $ make -C freedom-maker dreamplug-image raspberry-image virtualbox-image
+    $ git clone https://github.com/freedombox/freedom-maker.git freedom-maker
+    $ make -C freedom-maker dreamplug raspberry virtualbox
```

Comment:
[Sunil Mohan Adapa]
> I think Alioth repository is still the primary repository.  GitHub
> pull requests were being accepted, however.  Both the repositories
> were up-to-date last I checked.  @petterreinholdtsen would know.

Before the freedombox project on github, I considered Alioth the
authorative source for freedom-maker, and my private copy a convenience
for people to send pull requests to.

I have not considered what we should consider the authorative source
now, but will keep using Alioth as the primary repo until someone
protests. :)

-- 
Happy hacking
Petter Reinholdtsen